app/admin/fiori-service.cds

```diff
@@ -96,7 +111,46 @@ annotate AdminService.GenreHierarchy with @UI: {
     LineItem               : [{
         $Type: 'UI.DataField',
         Value: name,
-    }]
+        Label : 'Genre'
+    }],
+};
+
+annotate AdminService.ContentsHierarchy with @UI: {
+    PresentationVariant  : {
+        $Type         : 'UI.PresentationVariantType',
+        RequestAtLeast: [name],
+        Visualizations: ['@UI.LineItem', ],
+    },
+    LineItem             : [{
+        $Type: 'UI.DataField',
+        Value: name,
+        },
+        {
+      $Type: 'UI.DataField',
+      Value: page,
+    }],
+    HeaderInfo            : {
+        $Type         : 'UI.HeaderInfoType',
+        TypeName      : 'Contents Level',
+        TypeNamePlural: 'Contents Levels',
```

Comment:
Use i18n keys here and also in the rest of this annotation to enable translation at least into german.